Modules/Loadable/Markups/MRML/vtkMRMLMarkupsAngleNode.h

```diff
@@ -63,6 +64,12 @@ class  VTK_SLICER_MARKUPS_MODULE_MRML_EXPORT vtkMRMLMarkupsAngleNode : public vt
   /// Get node XML tag name (like Volume, Model)
   const char* GetNodeTagName() override {return "MarkupsAngle";}
 
+  /// Get markup name
+  const char* GetMarkupName() override {return "Angle";};
+
+  /// Get markup short name
+  const char* GetMarkupShortName() override {return "A";};
```

Comment:
A name can be shortened for many reasons. It would be better to have a method name that tells more about its intent.
If it is for choosing node name prefix then the method could be `GetDefaultNodeNamePrefix`.